main.go

```diff
@@ -177,3 +178,14 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func setLoggingLevel(debug bool, logLevel string) {
+	// the debug flag takes precedence over the loglevel flag
+	if debug {
+		log.SetLevel(log.DebugLevel)
+	} else {
+		level, err := log.ParseLevel(logLevel)
+		setupLog.Error(err, "unable to parse loglevel", "loglevel", logLevel)
```

Comment:
Forgot that I had one question, @rumstead:

Should this be:
```
if err != nil {
  // log error about unable unable to parse loglevel?
} else {
  set level?
}
```

Reply:
Yes, good catch! I committed as the wrong user and had a heck of a time undoing them.